README.md

```diff
@@ -74,8 +74,93 @@ require_once __DIR__ . DIRECTORY_SEPARATOR . 'vendor' . DIRECTORY_SEPARATOR . 'a
 $runner->addExtension(new \mageekguy\atoum\blackfire\extension());
 ```
 
+## Other examples
+
+### Define custom metrics
+
+```php
+$this
+    ->blackfire()
+        ->defineMetric(new \Blackfire\Profile\Metric("example_method_calls", "=Example::method"))
```

Comment:
or better, i'll update this PR to do a `->blackfire($this->getBlackfireClient())` in the examples. And after use `->blackfire` in PR #3 examples.